jscomp/others/belt_MapInt.mli

```diff
@@ -1,133 +1,151 @@
+val some: 'v t -> (key -> 'v -> bool) -> bool
 (** [some m p] checks if at least one binding of the map
-    satisfy the predicate [p].
- *)
+    satisfy the predicate [p]. Order unspecified *)
+
 val size: 'v t -> int
+
 val toList: 'v t -> (key * 'v) list
-(** In increasing order with respect *)
+(** In increasing order. *)
+
 val toArray: 'v t -> (key * 'v) array
-val ofArray: (key * 'v) array -> 'v t     
+
+val ofArray: (key * 'v) array -> 'v t
 [@@ocaml.deprecated "Use fromArray instead"]
-val fromArray: (key * 'v) array -> 'v t     
-val keysToArray: 'v t -> key array 
+
+val fromArray: (key * 'v) array -> 'v t
+
+val keysToArray: 'v t -> key array
+
 val valuesToArray: 'v t -> 'v array
-val minKey: _ t -> key option 
+
+val minKey: _ t -> key option
+
 val minKeyUndefined: _ t -> key Js.undefined
+
 val maxKey: _ t -> key option
+
 val maxKeyUndefined: _ t -> key Js.undefined
+
 val minimum: 'v t -> (key * 'v) option
+
 val minUndefined: 'v t -> (key * 'v) Js.undefined
+
 val maximum: 'v t -> (key * 'v) option
+
 val maxUndefined: 'v t -> (key * 'v) Js.undefined
+
 val get: 'v t -> key -> 'v option
+
 val getUndefined: 'v t -> key -> 'v Js.undefined
-val getWithDefault:  'v t -> key -> 'v  -> 'v
-val getExn: 'v t -> key -> 'v 
+
+val getWithDefault: 'v t -> key -> 'v -> 'v
+
+val getExn: 'v t -> key -> 'v
 
 val checkInvariantInternal: _ t -> unit
 (**
    {b raise} when invariant is not held
 *)  
 
-(****************************************************************************)
+val checkInvariantInternal: _ t -> unit
+(**
+   {b raise} when invariant is not held
+*)
 
-val remove: 'v t ->  key -> 'v t
+val remove: 'v t -> key -> 'v t
 (** [remove m x] returns a map containing the same bindings as
    [m], except for [x] which is unbound in the returned map. *)
+
 val removeMany: 'v t -> key array -> 'v t
 
-val set: 'v t ->  key -> 'v -> 'v t
+val set: 'v t -> key -> 'v -> 'v t
 (** [add m x y] returns a map containing the same bindings as
    [m], plus a binding of [x] to [y]. If [x] was already bound
    in [m], its previous binding disappears. *)
 
-val updateU: 'v t -> key -> ('v option -> 'v option [@bs]) -> 'v t 
-val update: 'v t -> key -> ('v option -> 'v option) -> 'v t 
-val mergeArray: 'v t -> (key * 'v) array -> 'v t
-    
+val updateU: 'v t -> key -> ('v option -> 'v option [@bs]) -> 'v t
+val update: 'v t -> key -> ('v option -> 'v option) -> 'v t
+
 val mergeU:
     'v t -> 'v2 t ->
     (key -> 'v option -> 'v2 option -> 'c option [@bs]) ->
     'c t
 val merge:
     'v t -> 'v2 t ->
     (key -> 'v option -> 'v2 option -> 'c option) ->
-    'c t      
+    'c t
 (** [merge m1 m2 f] computes a map whose keys is a subset of keys of [m1]
     and of [m2]. The presence of each such binding, and the corresponding
     value, is determined with the function [f].
  *)
 
-val keepU: 
-    'v t -> 
-    (key -> 'v -> bool [@bs]) -> 
+val mergeMany: 'v t -> (key * 'v) array -> 'v t
```

Comment:
we should deprecate `mergeArray` and add this one to avoid some potential breakage